Update my site dashboard cards styles

ravishanker opened the pull request:
This PR updates the MySite dashboard card styles for Stats card, Posts card, Prompts card
- Remove icon in the title
- Update font to sans-serif
- Update cardCornerRadius to 10dp
- Update Jetpack logo


<img width=320 src="https://user-images.githubusercontent.com/990349/224586948-18a0620e-ae72-4512-add1-e5eaf37a1849.png" />


To test:

- Launch WP or JP app
- Go to MySite and select Home
- Verify that the Stats, Posts and Prompts cards now do not show icon in the title, and font is sans-serif

>**Note**
> **Work on a draft post** on Android has **Go to drafts** action link on the card at the bottom

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

ravishanker committed:
Update Stats card

Remove icon and update font to sans-serif

ravishanker committed:
Update Blogging Prompts card

Remove icon and update font to sans-serif

ravishanker committed:
Update Posts card

Remove icon and update font to sans-serif

ravishanker committed:
Update font to sans-serif

ravishanker committed:
Update Jetpack logo

osullivanchris reviewed (approved):
😎  Looking much better, thanks @ravishanker 

Some possible enhancements:
- the title for Blogging Prompts is different to the other small card titles. Could it be made the same?
- What do you think about making the corner radius of the quick actions and cards larger. E.g. 10dp. Modern material design tends to be more roundy. I think it might look better. But if it feels wrong to you, let me know.
- There was another idea in the designs to remove the 'Go to stats' and 'Go to drafts' links at the bottom of the card, and instead colorise the word Stats and Drafts in the title of the card, to show that the user can access all the content there.

exported the Figma design below so you can see what I mean re; links in titles, and larger corner radii 

<img width="1958" alt="Hack Week changes Android" src="https://user-images.githubusercontent.com/50576199/224295916-31d05726-a77e-4452-bb6d-4631566b5499.png">

ravishanker committed:
Update Prompts title to match other cards title

ravishanker committed:
Update MySite cards styles

Update card styles of Quick Links and MySite Dashboard cards
- cardCornerRadius to 10dp
- font to sans-serif

ravishanker commented:
> Some possible enhancements:
> 
>     * the title for Blogging Prompts is different to the other small card titles. Could it be made the same?
> 
>     * What do you think about making the corner radius of the quick actions and cards larger. E.g. 10dp. Modern material design tends to be more roundy. I think it might look better. But if it feels wrong to you, let me know.
> 
>     * There was another idea in the designs to remove the 'Go to stats' and 'Go to drafts' links at the bottom of the card, and instead colorise the word Stats and Drafts in the title of the card, to show that the user can access all the content there.
> 
👋 @osullivanchris 
- Updated the Prompts title to match other card titles 
- Updated Card corner radius to 10dp

Removing 'Go to stats', 'Go to drafts', is a bit tricky due to localization unless it is OK to make the entire title a hyper link! Either way, this needs to be done separately.

ravishanker committed:
Revert "Update Jetpack logo"

This reverts commit b68730d689dfe22422b0ef152f0d4789f0784713.

ravishanker committed:
Update Jetpack splash logo

osullivanchris commented:
> Updated the Prompts title to match other card titles. Updated Card corner radius to 10dp

ah yes! Looks much better! glad we did this

> Removing 'Go to stats', 'Go to drafts', is a bit tricky due to localization unless it is OK to make the entire title a hyper link! Either way, this needs to be done separately.

ok understood. No problem to park this one for now.
